### PR TITLE
Use local epel mirrorlist

### PR DESCRIPTION
**Pull request by djgalloway**
Related to:
http://tracker.ceph.com/issues/12739
http://tracker.ceph.com/issues/12778

### CODE REVIEW

**Comment by ktdreyer** [comment]
I didn't know you could put newlines in like that?

**Comment by djgalloway** [comment]
Tested it on plana11.  See [[repository] OPTIONS](http://linux.die.net/man/5/yum.conf) section.

The other way to maintain this would be to create separate local files containing mirror URLs

File: roles/common/defaults/main.yml [hunk 1]
```diff
@@ -9,20 +9,17 @@ rhsm_repos: []
 kerberos_realm: EXAMPLE.COM
 
 epel_mirror_baseurl: "http://dl.fedoraproject.org/pub/epel"
-epel_mirrorlist: "http://mirrors.fedoraproject.org"
 epel_repos:
   epel:
     name: "Extra Packages for Enterprise Linux"
-    baseurl: "{{ epel_mirror_baseurl }}/{{ ansible_distribution_major_version }}/$basearch"
-    mirrorlist: "{{ epel_mirrorlist }}/metalink?repo=epel-{{ ansible_distribution_major_version }}&arch=$basearch"
+    baseurl: "{{ epel_mirror_baseurl }}/{{ ansible_distribution_major_version }}/$basearch\nhttp://mirrors.cat.pdx.edu/epel/{{ ansible_distribution_major_version }}/$basearch\nhttp://mirror.pnl.gov/epel/{{ ansible_distribution_major_version }}/$basearch\nhttp://fedora-epel.mirror.lstn.net/{{ ansible_distribution_major_version }}/$basearch"
```
Comment (andrewschoen) [inline roles/common/defaults/main.yml]: Could we instead have ansible manage a file, put that file on the remote and then point to it here in baseurl?

All this put into one string is difficult to parse, IMHO.

Comment (ktdreyer) [inline roles/common/defaults/main.yml]: Agreed, a separate file sounds a lot saner

Comment (zmc) [inline roles/common/defaults/main.yml]: A template might be useful here:
http://docs.ansible.com/ansible/template_module.html

**Comment by djgalloway** [comment]
Suggestions on a cleaner way to do this are welcome

File: roles/common/tasks/epel.yml [hunk 2]
```diff
@@ -16,6 +16,12 @@
   register: epel_repo
   with_dict: epel_repos
 
+- name: Configure local epel mirrorlists
+  template: src={{ item }} dest=/etc/yum.repos.d/{{ item }} owner=root group=root mode=0644
```
Comment (andrewschoen) [inline roles/common/tasks/epel.yml]: nitpick: I prefer this kind of formatting:

```
template:
  src: "{{ item }}"
  dest: "/etc/yum.repos.d/{{ item }}"
...
```

I think it's easier to read and makes for better diffs if/when we ever change this code.

**Comment by andrewschoen** [comment]
All I had was a style nitpick, everything else here looks good to me.

**Comment by ktdreyer** [comment]
thanks @djgalloway !

**Pull request merged**